File: ws/tipan/U2/Ws15Inheritance/ZooApp/src/main/java/ec/edu/espe/zooapp/model/Vaccine.java


package ec.edu.espe.zooapp.model;

/**
 *
 * @author Lisbeth Tipan
 */
public class Vaccine {
    private int id;
    private String description;

    public Vaccine(int id, String description) {
        this.id = id;
        this.description = description;
    }

    @Override
    public String toString() {
        return "Vaccine{" + "id=" + id + ", description=" + description + '}';
    }
    
    /**
     * @return the id
     */
    public int getId() {
        return id;
    }

    /**
     * @param id the id to set
     */
    public void setId(int id) {
        this.id = id;
    }

    /**
     * @return the description
     */
    public String getDescription() {
        return description;
    }

    /**
     * @param description the description to set
     */
    public void setDescription(String description) {
        this.description = description;
    }
    
}

File: hw/pilatasig/u2/ZooApp JavaScript/ec/edu/espe/zoo/model/Vaccine.js
export class Vaccine {
    constructor(id, description) {
        this._id = id; 
        this._description = description;
    }

    
    get description() {
        return this._description;
    }

    
    set description(newDescription) {
        this._description = newDescription;
    }

    
    get id() {
        return this._id;
    }

    
    set id(newId) {
        this._id = newId;
    }

    toString() {
        return `Vaccine{id=${this._id}, description=${this._description}}`;
    }
}

